docs/log_collection.md

```diff
@@ -1,5 +1,8 @@
 # Android Log Collection
 
+{{< site-region region="us3,gov" >}}
+<div class="alert alert-warning">Android log collection is not supported for this site.</div>
+{{< /site-region >}}
```

Comment:
Hi, why is Log Collection not supported on those sites?

Reply:
Hi @xgouchet! We received confirmation from @pgkamat a few weeks back to add notes that log collection is not supported for US3 in its entirety as there was a Jira issue pertaining to this. It looks like gov is listed here, we will update and remove.

Reply:
Hi @xgouchet, just updated and added the gov code block.